server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:0.10
+
+RUN \
+  useradd -d /app -m oam
+
+USER oam
```

Comment:
Na, unfortunately, the user has to be created before referencing it with `USER`.